### PR TITLE
Mon 4790 add missing fields

### PR DESCRIPTION
**Pull request by victorvassilev**
## Description

Add extra fields to host and service listing via api v2

**Fixes** # 4790

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Run REST calls towards monitoring/services and monitoring/hosts and check payload

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [x] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: src/Centreon/Domain/Monitoring/Host.php [hunk 1]
```diff
@@ -311,6 +311,18 @@ class Host
...
 
+    /**
+     * @Serializer\Groups({"host_main","host_full"})
+     * @var int|null
+     */
+    private $scheduledDowntimeDepth;
+
+    /**
+     * @Serializer\Groups({"host_main", "host_full"})
+     * @var int
```
Comment (callapa) [inline src/Centreon/Domain/Monitoring/Host.php]:
```suggestion
     * @var int|null
```

File: src/Centreon/Domain/Monitoring/Host.php [hunk 2]
```diff
@@ -1146,4 +1158,40 @@ public function setTimezone(?string $timezone): Host
...
+     * @return Host
+     */
+    public function setScheduledDowntimeDepth(?int $scheduledDowntimeDepth): Host
+    {
+        $this->scheduledDowntimeDepth = $scheduledDowntimeDepth;
+        return $this;
+    }
+
+    /**
+     * @return int
```
Comment (callapa) [inline src/Centreon/Domain/Monitoring/Host.php]:
```suggestion
     * @return int|null
```

File: src/Centreon/Domain/Monitoring/Host.php [hunk 3]
```diff
@@ -1146,4 +1158,40 @@ public function setTimezone(?string $timezone): Host
...
+     */
+    public function getCriticality(): ?int
+    {
+        return $this->criticality;
+    }
+
+    /**
+     * @param int $criticality
```
Comment (callapa) [inline src/Centreon/Domain/Monitoring/Host.php]:
```suggestion
     * @param int|null $criticality
```

File: src/Centreon/Domain/Monitoring/Service.php [hunk 4]
```diff
@@ -825,4 +831,22 @@ public function setStateType(int $stateType): Service
         $this->stateType = $stateType;
         return $this;
     }
+
+    /**
+     * @return int
```
Comment (callapa) [inline src/Centreon/Domain/Monitoring/Service.php]:
```suggestion
     * @return int|null
```

File: src/Centreon/Domain/Monitoring/Service.php [hunk 5]
```diff
@@ -825,4 +831,22 @@ public function setStateType(int $stateType): Service
...
+     */
+    public function getCriticality(): ?int
+    {
+        return $this->criticality;
+    }
+
+    /**
+     * @param int $criticality
```
Comment (callapa) [inline src/Centreon/Domain/Monitoring/Service.php]:
```suggestion
     * @param int|null $criticality
```

**Comment by callapa** [comment]
Don't forget to generate the HTML documentation.
Download https://github.com/Redocly/redoc (use nodeJS)
and lunch command:
`node cli/index.js bundle --options.hideDownloadButton=true centreon-api-v2.yaml -o centreon-api-v2.html`

Before doing this, please rebase from the master to avoid conflicts with the yaml file.

**Comment by kduret** [comment]
just missing html documentation.
Guide is here : https://github.com/centreon/centreon/pull/8309/files#diff-e36709718fadef4ef0aa8a8483d5496dR1

**Pull request merged**